### PR TITLE
[CI:DOCS] Fix remote docs

### PR DESCRIPTION
**Pull request by ashley-cui**
Fix renaming bug in remote-docs.sh
Remove mentions of 'remote' in windows and mac dos
Remove podman-remote.conf.5

Closes https://github.com/containers/libpod/issues/6584

Signed-off-by: Ashley Cui <acui@redhat.com>

### CODE REVIEW

**Comment by jwhonce** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming happy tests

File: docs/source/markdown/podman-remote.1.md [hunk 1]
```diff
@@ -132,14 +132,3 @@ the exit codes follow the `chroot` standard, see below:
 | [podman-unpause(1)](podman-unpause.1.md)         | Unpause one or more containers.                                             |
 | [podman-version(1)](podman-version.1.md)         | Display the Podman version information.                                     |
 | [podman-volume(1)](podman-volume.1.md)           | Manage Volumes.                                                             |
-
-## FILES
```
Comment (rhatdan) [inline docs/source/markdown/podman-remote.1.md:136]: Could you replace this with containers.conf

Comment (rhatdan) [reply docs/source/markdown/podman-remote.1.md:136]: We are going to need the containers.conf man page or a slimmed down one, along with the containers-storage.conf and containers-image.conf man pages.

@vrothberg Back in the day we could not maintain these in vendor directory, is this still true?

**Comment by rhatdan** [comment]
/approve
/lgtm
I am going to approve for now.  @ashley-cui lets open another PR to add information to the podman remote command to explain the containers.conf file.

**Comment by ashley-cui** [comment]
> I am going to approve for now. @ashley-cui lets open another PR to add information to the podman remote command to explain the containers.conf file.

@rhatdan Sounds good, though you may have to explain what ends up in containers.conf to me first. I looked through and haven't seen any docs on containers.conf?

**Pull request merged**